ArduCopter/tuning.cpp

```diff
@@ -184,10 +184,14 @@ void Copter::tuning()
         attitude_control->get_rate_yaw_pid().filt_E_hz(tuning_value);
         break;
 
-    case TUNING_SYSTEM_ID_MAGNITUDE:
 #if MODE_SYSTEMID_ENABLED == ENABLED
+    case TUNING_SYSTEM_ID_MAGNITUDE:
```

Comment:
I think the #if should remain after the case otherwise we will see a compile warning (or perhaps even a compile failure) if MODE_SYSTEMID_ENABLED is not ENABLED.

Reply:
I have reverted this. There are another couple above that maybe should be done in the same way. Should we change that while we are here?

Reply:
@lthall, yes probably..txs